cartographer_ros/cartographer_ros/node_main.cc

```diff
@@ -38,14 +41,20 @@ void Run() {
   tf2_ros::TransformListener tf(tf_buffer);
   NodeOptions node_options;
   TrajectoryOptions trajectory_options;
-  std::tie(node_options, trajectory_options) =
-      LoadOptions(FLAGS_configuration_directory, FLAGS_configuration_basename);
-
+  if (FLAGS_start_trajectory_with_default_topics) {
+    std::tie(node_options, trajectory_options) = LoadOptions(
+        FLAGS_configuration_directory, FLAGS_configuration_basename);
+  } else {
+    node_options = LoadNodeOptions(FLAGS_configuration_directory,
```

Comment:
I saw another _LoadOptions()_ in `start_trajectory_main.cc`. It seems like returning TrajectoryOptions. I think we may want to clean up around LoadOptions later? For now, I am thinking of just adding the if statement on _StartTrajectoryWithDefaultTopics_(line 55). How does it sound?